### PR TITLE
fix repos upgrade integration

### PR DESCRIPTION
**Pull request by jsingle**
@zacharym these tests needed some fixin

### CODE REVIEW

**Comment by jsingle** [comment]
OOps broke other stuff

**Comment by zacharym** [comment]
going to wait for this to pass, then i'll take a look

File: tests/integration/cli/repos_test.py [hunk 1]
```diff
@@ -24,19 +24,19 @@ def setUp(self):
         os.chmod(self.fake_override_dir, 0777)
         os.chmod(self.fake_from_dir, 0777)
 
-        self.old_repos_dir = constants.REPOS_DIR
-        constants.REPOS_DIR = self.temp_repos_dir
         self.fake_override_repo_location = os.path.join(self.fake_override_dir, 'fake-repo')
         self._set_up_fake_local_repo(path=self.fake_override_repo_location)
         self.fake_from_repo_location = os.path.join(self.fake_from_dir, 'fake-repo')
         self._set_up_fake_local_repo(path=self.fake_from_repo_location)
+        self.fake_source_repo_location = '/tmp/fake-repo'
```
Comment (zacharym) [inline tests/integration/cli/repos_test.py]: is there a reason why we are stuck using /tmp instead of mkdtemp ? I seem to recall there being something why

**Comment by zacharym** [comment]
seems legit

File: tests/integration/cli/repos_test.py (hunk 1 above)
Comment (jsingle) [inline tests/integration/cli/repos_test.py]: The code could in theory be better.  But the specs fixture needs to point to a defined place, not something that we can dynamically create in the testcase.  at least with the current pattern

**Pull request merged**